### PR TITLE
Security issue

### PR DESCRIPTION
**Pull request by hemant7405**
- Improvement: Hardened the security of the plugin.
- Fix: Reset functionality is not working in the backend settings.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -316,11 +328,11 @@ public function submit_color() {
 					print esc_attr( 'Sorry, your nonce did not verify.' );
 					exit;
 				} else {
-					$snippet_box_bg   = esc_attr( $_POST['snippet_box_bg'] );
-					$snippet_title_bg = esc_attr( $_POST['snippet_title_bg'] );
-					$border_color     = esc_attr( $_POST['snippet_border'] );
-					$title_color      = esc_attr( $_POST['snippet_title_color'] );
-					$box_color        = esc_attr( $_POST['snippet_box_color'] );
+					$snippet_box_bg   = esc_attr( sanitize_text_field( $_POST['snippet_box_bg'] ) );
```
Comment (patilvikasj) [inline index.php:331]: No need to escape and sanitize POST variable, Just sanitize will work.

File: init.php [hunk 2]
```diff
@@ -506,7 +506,7 @@ public function save( $post_id ) {
 				$name = esc_attr( $field['id'] ) . '_id';
 				$old  = get_post_meta( $post_id, $name, ! $field['multiple'] /* If multicheck this can be multiple values */ );
 				if ( isset( $field['save_id'] ) && $field['save_id'] ) {
-					$new = isset( $_POST[ $name ] ) ? esc_attr( $_POST[ $name ] ) : null;
+					$new = isset( $_POST[ $name ] ) ? esc_attr( sanitize_text_field( $_POST[ $name ] ) ) : null;
```
Comment (patilvikasj) [inline init.php:509]: Same here. Do not sanitize and escape variable. Just sanitize will work.

File: index.php (hunk 1 above)
Comment (hemant7405) [reply index.php]: changes have done.

File: init.php (hunk 2 above)
Comment (hemant7405) [reply init.php]: Changes have done.

**Review by patilvikasj — APPROVED** [review]

**Pull request merged**